c++/src/capnp/compat/json.h

```diff
@@ -85,6 +83,11 @@ class JsonCodec {
   // setHasMode(HasMode::NON_DEFAULT) to specify that default-valued primitive fields should be
   // omitted as well.
 
+  void setIgnoreUnrecognizedFields(bool ignore);
```

Comment:
Hmm default-true options can be a bit confusing. What do you think of inverting this to `setRejectUnknownFields()`?

Also, maybe bikeshedding, but there is precedent (going back to protobuf) for calling these "unknown fields" rather than "unrecognized fields".

Reply:
Yes, I agree with both points, done.

The reason why this was named `setAccept...` in the first place was due to building it inside-out, where I avoided a double negative at the actual check ("require that not reject is true" sounded wrong). Naturally, I ended up with a boolean that defaulted to true, and hence the option as wrapper for it. From an API perspective, I agree that the default value false/0 should take precedence to avoid confusion.

Regarding unrecognized vs. unknown: I followed the wording in a comment of the original author. However, *unknown* seems to be the preferred term in most of Cap'n Protos codebase, and given its precedent in Protobuf, switching out the adverb seems to be the more natural solution here.